outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "site_bucket" {
+  value = "${aws_s3_bucket.site_bucket.id}"
+}
+
+output "logs_bucket" {
```

Comment:
The `output` here is `logs_bucket` (plural), while the associated log variable is `log_bucket_name` (singular). I think it would be good to make both singular or both plural.